platform/darwin/src/MGLLineStyleLayer.h

```diff
@@ -366,6 +366,50 @@ MGL_EXPORT
 */
 @property (nonatomic) MGLTransition lineGapWidthTransition;
 
+#if TARGET_OS_IPHONE
+/**
+ Defines a gradient with which to color a line feature. Can only be used with
+ GeoJSON sources that specify `"lineMetrics": true`.
+ 
+ This property is only applied to the style if `lineDasharray` is set to `nil`,
```

Comment:
This should say `lineDashPattern`. The codegen script isn’t aliasing `lineDasharray` for some reason.

Reply:
#13000